server/util/networking/networking.go

```diff
@@ -441,7 +441,11 @@ func SetupVethPair(ctx context.Context, netNamespace, vmIP string, vmIdx int) (_
 			return nil, err
 		}
 		cleanupStack = append(cleanupStack, func(ctx context.Context) error {
-			return runCommand(ctx, "ip", "route", "delete", cloneIP)
+			err := runCommand(ctx, "ip", "route", "delete", cloneIP)
+			if err != nil && !strings.Contains(err.Error(), "No such process") {
+				return err
+			}
+			return nil
 		})
 	} else {
 		log.Debugf("ip route %s via %s already exists", cloneIP, cloneEndpointAddr)
```

Comment:
The "routeExists" logic seems a little sketchy since we don't expect these routes to be used by anything other than VMs. Maybe we should promote this log line to Warning level or return an Unavailable error here?

Reply:
I checked and we haven't seen this log for over 2 weeks in dev - upgraded to an error